src/api/app/views/webui/project/_edit.html.haml

```diff
@@ -14,3 +14,10 @@
     %a.cancel.btn.btn-outline-danger.px-4
       Cancel
     = submit_tag('Update', class: 'btn btn-primary px-4')
+
+:javascript
+  $(document).ready(function() {
+    $('.cancel').on('click', function(){
+      location.reload();
```

Comment:
Hm, I don't understand. Why?

Reply:
This was a first attempt, but in the next commit this is replaced by a normal `remote: true` link to use UJS.